lib/rouge/themes/github.rb

```diff
@@ -6,68 +6,139 @@ module Themes
+        palette :comment     => P_GRAY_3[@mode]
+        palette :constant    => P_BLUE_2[@mode]
+        palette :entity      => P_PURPLE_2[@mode]
+        palette :heading     => P_BLUE_5[@mode]
+        palette :keyword     => P_RED_3[@mode]
+        palette :string      => P_BLUE_1[@mode]
+        palette :tag         => P_GREEN_1[@mode]
+        palette :variable    => P_ORANGE_2[@mode]
+
+        palette :fgDefault   => P_GRAY_1[@mode]
+
+        palette :fgInserted  => P_GREEN_0[@mode]
+        palette :bgInserted  => P_GREEN_8[@mode]
+
+        palette :fgDeleted   => P_RED_0[@mode]
+        palette :bgDeleted   => P_RED_8[@mode]
+
+        palette :fgError     => P_GRAY_0[@mode]
+        palette :bgError     => P_RED_7[@mode]
+      end
+
+      def self.make_light!
+        palette :comment     => P_GRAY_5[@mode]
+        palette :constant    => P_BLUE_6[@mode]
+        palette :entity      => P_PURPLE_5[@mode]
+        palette :heading     => P_BLUE_6[@mode]
+        palette :keyword     => P_RED_5[@mode]
+        palette :string      => P_BLUE_8[@mode]
+        palette :tag         => P_GREEN_6[@mode]
+        palette :variable    => P_ORANGE_6[@mode]
+
+        palette :fgDefault   => P_GRAY_9[@mode]
+
+        palette :fgInserted  => P_GREEN_6[@mode]
+        palette :bgInserted  => P_GREEN_0[@mode]
+
+        palette :fgDeleted   => P_RED_7[@mode]
+        palette :bgDeleted   => P_RED_0[@mode]
+
+        palette :fgError     => P_GRAY_0[@mode]
+        palette :bgError     => P_RED_7[@mode]
+      end
+
+      light!
+
+      style Keyword,                    :fg => :keyword
+
+      style Generic::Error,             :fg => :fgError
+
+      style Generic::Deleted,           :fg => :fgDeleted, :bg => :bgDeleted
+
+      style Name::Builtin,
+            Name::Class,
+            Name::Constant,
+            Name::Namespace,            :fg => :variable
+
+      style Literal::String::Regex,
+            Name::Attribute,
+            Name::Tag,                  :fg => :tag
+
+      style Generic::Inserted,          :fg => :fgInserted, :bg => :bgInserted
+
+      style Keyword::Constant,
+            Literal,
+            Literal::String::Backtick,
+            Name::Builtin::Pseudo,
+            Name::Exception,
+            Name::Label,
+            Name::Property,
+            Name::Variable,
+            Operator,                   :fg => :constant
+
+      style Generic::Heading,
+            Generic::Subheading,        :fg => :heading, :bold => true
+
+      style Literal::String,            :fg => :string
+
+      style Name::Decorator,
+            Name::Function,             :fg => :entity
+
+      style Error,                      :fg => :fgError, :bg => :bgError
+
+      style Comment,
+            Generic::Lineno,
+            Generic::Traceback,         :fg => :comment
+
+      style Name::Entity,
+            Literal::String::Interpol,
+            Text,                       :fg => :fgDefault
```

Comment:
**suggestion:** Could we add a background colour to distinguish between light and dark themes? FYI, we [use](https://github.com/rouge-ruby/rouge/blob/master/lib/rouge/theme.rb#L112) styles on `Text` token to set the general foreground and background colour.

```
Text, :fg => :fgDefault, :bg => :bgDefault
```